src/components/RsvpResponse.js

```diff
@@ -62,11 +43,6 @@ const RsvpResponse = () => {
 
 	Listener({ setRsvpStatus }, getFromGlobal('post_id'));
 
-	// Make sure rsvpStatus is a valid status, if not, set to default.
-	if (!items.some((item) => item.value === rsvpStatus)) {
-		setRsvpStatus(defaultStatus);
-	}
-
```

Comment:
Looks like this file got messed up. These changes should remain. Think something happened when you updated from `main` after I pushed code. Looks like the file can just be reverted to what's on `main`.